src/components/ContractTransaction.tsx

```diff
@@ -44,15 +44,7 @@ const ContractTransaction = (props: {
                     props.address.address === signer().address)
             }
             fallback={
-                <Show
-                    when={!allowAnyAddress()}
-                    fallback={
-                        <ConnectWallet
-                            derivationPath={props.address.derivationPath}
-                        />
-                    }>
-                    <ConnectAddress address={props.address} />
```

Comment:
Can we get rid of that? For reverse swap claim transactions we have to connect to the address we specified on swap creation?